apps/central-scan/frontend/src/api.ts

```diff
@@ -74,3 +74,17 @@ export const logOut = {
     });
   },
 } as const;
+
+export const deleteBatch = {
+  useMutation() {
+    const apiClient = useApiClient();
+    const queryClient = useQueryClient();
+    return useMutation(apiClient.deleteBatch, {
+      async onSuccess() {
+        // Because we poll auth status with high frequency, this invalidation isn't strictly
+        // necessary
+        await queryClient.invalidateQueries(getAuthStatus.queryKey());
```

Comment:
Do we need to invalidate auth status after deleting a batch, or was this just accidentally left during copy/paste?

Reply:
I don't know. Probably not, given the action being performed. I'll assume not unless @arsalansufi tells me otherwise.

Reply:
Nope, no need to invalidate auth status here!